spec/models/green_lanes/category_assessment_spec.rb

```diff
@@ -74,6 +74,7 @@
 
       it { is_expected.to be_an Array }
       it { is_expected.to all be_instance_of described_class }
+      it { is_expected.to all be_frozen }
```

Comment:
This makes me think the second `tap` works on the categories, not the IDs.
but why do we need to freeze them?

Reply:
They are not intended to be mutatable data - so this avoids developers mutating them by mistake and introducing hard to undestand bugs

Took 5 minutes to add but could save us a load of debugging in the future